chat-components/src/components/outofofficehourspane/OOOHPane.tsx

```diff
@@ -26,15 +26,15 @@ function OOOHPane(props: IOOOHPaneProps) {
                     id={elementId}
                     tabIndex={-1}
                     styles={containerStyles}
-                    role={props.controlProps?.role}
+                    role="alert"
                     className={props.styleProps?.classNames?.containerClassName}
                     dir={props.controlProps?.dir ?? defaultOOOHPaneControlProps.dir}>
 
                     {!props.controlProps?.hideTitle && (decodeComponentString(props.componentOverrides?.title) ||
                         <Label
                             className={props.styleProps?.classNames?.titleClassName}
                             styles={titleStyles}
-                            tabIndex={-1}
+                            tabIndex={1}
```

Comment:
Also I do not think it is a good practice for non-interactive elements to have positive tabIndex